indico/modules/categories/templates/management/content.html

```diff
@@ -1,9 +1,69 @@
 {% extends 'categories/management/base.html' %}
 
+{% macro category_row(category) %}
+    <tr class="i-table">
+        <td class="i-table column-icon">
+            <a class="i-button-icon icon-handle"></a>
+        </td>
+        <td class="i-table column-icon">
+            <input type="checkbox"></input>
+        </td>
+        <td class="i-table column-icon">
+            {% if category.is_protected %}
```

Comment:
`is_self_protected`; and i'd show an open lock (like in the contribution list, AFAIK it's also red/green there) if `is_public`